apps/api/src/player/player.service.ts

```
@@ -0,0 +1,44 @@
+        { [`members.recruiter.${playerUuid}`]: { $exists: true } },
+        { [`members.recruit.${playerUuid}`]: { $exists: true } },
+      ],
+    });
+
+    if (!guild) {
+      return null;
+    }
+
+    let playerRank = '';
+    for (const rank of ['owner', 'chief', 'strategist', 'captain', 'recruiter', 'recruit']) {
+      if (guild.members[rank] && guild.members[rank][playerUuid]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)